src/components/transactions/ClaimRewards/ClaimRewardsModalContent.tsx

```diff
@@ -33,7 +33,7 @@ export const ClaimRewardsModalContent = () => {
   const { gasLimit, mainTxState: claimRewardsTxState, txError } = useModalContext();
   const { user, reserves } = useAppDataContext();
   const { currentChainId, currentMarketData, currentMarket } = useProtocolDataContext();
-  const { chainId: connectedChainId } = useWeb3Context();
+  const { chainId: connectedChainId, mockAddress } = useWeb3Context();
```

Comment:
Since we're using the terminology `watch only` instead of `mock`, please rename this variable to be consistent throughout the code changes - `watchOnlyModeAddress` suffices